src/components/Die/Die.js

```diff
@@ -31,39 +28,89 @@ import "./Die.scss";
 
 const Die = ({ die }) => {
   // You could extract these code bits and have a function which returns the style. LOOK ABOVE 1.
-  const dieImg = React.useMemo(() => {
+  const getDiceStyle = React.useMemo(() => {
+    const baseDieStyle = {
+      backgroundPosition: "center",
+      backgroundSize: "cover",
+      backgroundRepeat: "no-repeat",
+    };
+
     switch (die) {
       case 1:
-        return die1;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die1 + ")",
+        });
       case 2:
-        return die2;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die2 + ")",
+        });
       case 3:
-        return die3;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die3 + ")",
+        });
       case 4:
-        return die4;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die4 + ")",
+        });
       case 5:
-        return die5;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die5 + ")",
+        });
       case 6:
-        return die6;
-
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die6 + ")",
+        });
       default:
-        return;
+        return Object.assign(baseDieStyle, {
+          backgroundImage: "url(" + Die1 + ")",
+        });
+      // OR
+      //return {
+      //   ...baseDieStyle,
+      //   backgroundImage: "url(" + die1 + ")",
+      // }
     }
   }, [die]);
 
-  const style = {
-    backgroundImage: "url(" + dieImg + ")",
-    backgroundPosition: "center",
-    backgroundSize: "cover",
-    backgroundRepeat: "no-repeat",
-  };
+  //******************************************* QUESTION ********************************************
+  // I do not understand how the suggested code will shorten the result, because I believe that I was doing almost the same thing.
+  //I'd appreciate if you can elaborate a bit.
+
+  //------------------------------- My codes ---------------------
```

Comment:
You are correct! I think I managed to trick myself into something that is not present in your code haha. Your approach was perfectly fine! I'm sorry 😓